scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -704,5 +704,17 @@ <h5 class="modal-title" id="exampleModalLabel">Edit synopsis</h5>
 $('[data-bs-toggle=sidebar-collapse]').click(function() {
   SidebarCollapse();
 });
+
+// Allows a Sanger recipient to notify the user who ordered a Sanger verification that it was a true positive
```

Comment:
Sorry, I'm missing the gain from checking for the user in Sanger recipients? If they are posed to set the validation status, they got the info somehow, didn't they? The Sanger recipient can conveniently be a function email box at the hospital, to which several lab users have access.